api/sds.proto

```diff
@@ -0,0 +1,26 @@
+syntax = "proto3";
+
+package envoy.api.v2;
+
+import "api/discovery.proto";
+import "api/tls_context.proto";
+
+import "google/api/annotations.proto";
+
+service SecretDiscoveryService{
+  rpc StreamSecrets(stream DiscoveryRequest)
+      returns (stream DiscoveryResponse) {
+  }
+
+  rpc FetchSecrets(DiscoveryRequest)
+      returns (DiscoveryResponse) {
+    option (google.api.http) = {
+      post: "/v2/discovery:secrets"
+      body: "*"
+    };
+  }
+}
+
+message Secret {
+  TlsCertificate tls_certificate = 1;
```

Comment:
Should we make this a oneof now since we might add other types of secrets in the future? Not sure if the caller would need to specify the type? 

Also, doesn't this need to include name so that Envoy can match up the response secret with the requested secret?